src/components/TextField/interfaces.ts

```diff
@@ -0,0 +1,8 @@
+export interface Props {
```

Comment:
I think that we may want to expose a few more props for our `TextArea`.

* `disabled?: boolean` (will set disabled and readOnly to true/false in the react-bootstrap component) 
* `isInvalid?: boolean` (will set `isInvalid` to true/false in react-bootstrap component)
* `size?: 'small' | 'large'` (will set `size` to `'sm' | 'lg'` in react-bootstrap component)
* `onChange?: function` (will set `onChange` in react-boostrap component)

Reply:
This has been addressed.